compiler/src/skfmt.sk

```diff
@@ -35,11 +35,17 @@ untracked fun main(): void {
         .short("i")
         .about("Inplace edit file(s), if specified"),
     )
+    .arg(
+      Cli.Arg::string("filename")
```

Comment:
FWIW clang-format calls this option `--assume-filename`

Reply:
:+1: 
Renamed!